limitador-server/script/kind-setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script deploys the environment described in
+# limitador-server/kubernetes/README.md in a local cluster using kind
+# (https://github.com/kubernetes-sigs/kind)
+
+SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
+CLUSTER_NAME=limitador
+LIMITADOR_NAMESPACE=default
+
+kind delete cluster --name ${CLUSTER_NAME}
+kind create cluster --name ${CLUSTER_NAME}
+
+# Deploy Redis
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/redis-service.yaml
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/redis-statefulset.yaml
+
+# Deploy Limitador
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-config-configmap.yaml
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-service.yaml
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/limitador-deployment.yaml
+
+# Deploy Kuard. In kind we don't use the proxy protocol, so we need to disable
+# the option in the Envoy config
+sed 's/use_proxy_proto: true/use_proxy_proto: false/g' "${SCRIPT_DIR}"/../kubernetes/kuard-envoy-config-configmap.yaml | kubectl -n ${LIMITADOR_NAMESPACE} apply -f -
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-service.yaml
+kubectl -n ${LIMITADOR_NAMESPACE} apply -f "${SCRIPT_DIR}"/../kubernetes/kuard-deployment.yaml
+
+# Wait for everything to be ready
+for deployment in $(kubectl -n ${LIMITADOR_NAMESPACE} get deploy -oname)
+do
+  kubectl -n ${LIMITADOR_NAMESPACE} wait --timeout=300s --for=condition=Available "$deployment"
```

Comment:
Don't know if the purpose of this line is to control kuard/limitador, or also redis, but bear in mind, that redis is a statefulset (not a deployment), so I guess that `kubectl get deploy` won't include redis availability.

Reply:
In this case it's fine because Limitador is not ready until it can connect to Redis.

Reply:
Perfect!